pkg/project/project.go

```diff
@@ -20,6 +21,7 @@ import (
 	"github.com/ActiveState/cli/internal/multilog"
 	"github.com/ActiveState/cli/internal/osutils"
 	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/prompt"
```

Comment:
Personally, I don't think we should do any prompting in this package. Currently, there is some prompting for secrets, but that is old code and I suggest that we shouldn't use that to justify adding more dependency on prompting. I suggest moving the prompting into a new package within the runbits directory. Generally, I'd imagine this function returning all projects and then the runbits package prompting based on those options. If that leaves this function fully or near fully useless, then it may be worth trying to remove it unless there are other callers needing it. @Naatan, please jump in if I'm suggesting a poor direction.

Reply:
I agree, prompt should ideally only be used from runners or runbits. Generally speaking anything that produces output would fall under this rule.

Reply:
Committed a new runbits package.